tutorials/CountAndCache.qmd

```diff
@@ -0,0 +1,128 @@
+---
+title: "How to Cache function calls and Get Started: Optimize!"
```

Comment:
This appears to somewhat duplicate the `.md` file?

Reply:
Quite easy: I do not yet have a good way to (a) render quarto on CI (I think that should work now soon-ish since I found some nice features) nor (b) how to properly cache that because tutorials take some time to render and (c) how these caches could be copied into the docs _before_ the docs are rendered. So for now the `.md` files are rendered from these `.qmd` files locally for now and both are checked in.

Reply:
One could probably change the docs-CI to run quarto upfront and have some artefact cache and such (not that since this PR I know how to run quarto render even on dev versions of Manopt), but it would take me probably 2-3 weeks to get that working in which I would not work on any other code.

Reply:
I see, then let's keep it duplicated. Do you want me to re-apply the .md suggestions here?

Reply:
I already just transferred them :)

Reply:
And in the long run you are right, for best of cases

* Quarto should run before Documenter in CI
* Quarto should have a cache so only changed tutorials should be rerun

PlutoStaticHTML for example managed the first , but I was never able to get the second (and it produced HTML), so at some point I might give this a try.